internal/js_library/v2/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "description": "runtime dependencies for js_library",
+    "devDependencies": {
+        "@babel/core": "^7.1.0",
+        "@babel/preset-env": "^7.1.0",
+        "commander": "^2.18.0",
+        "mkdirp": "^0.5.1"
```

Comment:
can you avoid this extra dependency? the function is trivial to write out and we don't like dependencies (this one has licensing issues IIRC)

Reply:
Removed this dependency and noticed that node 10.12 (the new LTS) provides this built-in, so just using that functionality now insread (and submitted a PR to upgrade the version: #392)